upgrade/v2.5/README.md

```diff
@@ -75,6 +75,10 @@ Make sure you have all the `IPPools`, `BGPPeers`, `GlobalFelixConfig`, and `Glob
 
 ## 4. Upgrade calico
 
+  > Note: If you are updating a [Canal](https://github.com/projectcalico/canal/tree/master/k8s-install)
+  deployment do not use the RBAC manifest in the following step, instead use the
+  appropriate Canal RBAC manifest.
```

Comment:
Can you add a link here to allow them to potentially find the appropriate Canal RBAC manifest? Just in case they don't have this and don't know where to find it.